src/main/java/com/exasol/projectkeeper/validators/changelog/LastReleasePomFileReader.java

```diff
@@ -0,0 +1,30 @@
+package com.exasol.projectkeeper.validators.changelog;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import com.exasol.projectkeeper.repository.GitRepository;
+import com.exasol.projectkeeper.repository.TaggedCommit;
+
+/**
+ * This class reads the pom file of the latest release on the current branch.
+ */
+public class LastReleasePomFileReader {
+    private static final Path POM_PATH = Path.of("pom.xml");
+
+    /**
+     * Read the content of the pom.xml file from the last release on the current branch.
+     * 
+     * @param projectDirectory projects root directory
+     * @return content of the pom file
+     */
+    public Optional<String> readLatestReleasesPomFile(final Path projectDirectory) {
```

Comment:
You already have a class with git magic (GitRepository). Why did you create another one?

Reply:
I wanted to make a cut between the two abstraction levels. The GitRepository contains general-purpose git magic. 
This class contains the logic for getting the pom file.
In addition, it's quite useful to have a vertical cut somewhere for testing. (otherwise, I would have to test it as on block)